locales/en/filters.json

```
@@ -390,5 +390,9 @@
 	{
 		"filter.257.name": "uBlock Origin – Badware risks",
 		"filter.257.description": "Filter for risky sites, warning users of potential threats."
+	},
+	{
+		"filter.258.name": "Block Outsider Intrusion into LAN",
```

Стоит помянуть в названии uBO, т.к. в оригинальном фильтре не указано.
Например, в другом их фильтре вот так:
`! Title: uBlock Origin – Badware risks`

```suggestion
  "name": "uBlock Origin – Block Outsider Intrusion into LAN",
```